Update game panel instructions (#42)
nanos-world/docs: docs/core-concepts/server-manual/game-panels.md

:::tip

> Installation files are open source at Falaxir GitHub: https://github.com/Falaxir/nanos-world_yolks-game-panels. You can also check wich features is supported.

Reviewer: ```suggestion
Installation files are open source at Falaxir GitHub: https://github.com/Falaxir/nanos-world_yolks-game-panels. You can also check which features are supported.
```

Then, go to your panel on the category `SERVICE MANAGEMENT` and click on `NESTS`.

> You can create a new `NEST` by clicking the create button or click on a current nest.

Reviewer: ```suggestion
You can create a new `NEST` by clicking the create button or selecting an existing nest.
```

ampinstmgr -q ADS01
```

:::info Note

> To run `ampinstmgr` commands, you need to run them as the user `amp`, you can switch to this user using this command : `sudo su -l amp`

Reviewer: ```suggestion
To run `ampinstmgr` commands, you need to run them as the user `amp`, you can switch to this user using the command: `sudo su -l amp`.
```
